man/en/d2df.3.txt

```diff
@@ -70,7 +70,7 @@ sv_master_interval <number>
 
 @h2 Technical options
 net_master_list <host>
-	To which master server address the data will be sent. You can specify more than one, for example mpms.doom2d.org:25665,deadsoftware.ru:25665
+	To which master server address the data will be sent. You can specify more than one, for example dfms.doom2d.org:25665,dfms2.doom2d.org:1005,terminalcorner.ru:25665
```

Comment:
Two samples are enough. Leave only `dfms.doom2d.org:25665` and `dfms2.doom2d.org:1005` here for now.